### ISSUE TITLE
Select marker

### ISSUE THREAD

**Issue by Limonlol** [icerockdev/moko-maps#5]
Need callback for select marker
Id for marker, and pass it in callback for select

**Comment by Alex009** [comment]
will be available in 0.2.0 via:
```kotlin
suspend fun addMarker(
    image: ImageResource, 
    latLng: LatLng, 
    rotation: kotlin.Float, 
    onClick: (() -> kotlin.Unit)?
): dev.icerock.moko.maps.Marker
```

**Comment by Alex009** [comment]
released in 0.2.0
